### PR TITLE
Grep for potential errors in standalone tests

### LINKED ISSUE #12474: Standalone tests can silently fail
## 🐛 Bug

The CI passes with a green check mark, but the logs clearly show that the standalone tests fail.

### To Reproduce

Can be seen on the[ latest commits on master](https://dev.azure.com/PytorchLightning/pytorch-lightning/_build/results?buildId=64144&view=logs&j=3afc50db-e620-5b81-6016-870a6976ad29&t=d9f671c5-a304-5675-5394-961fd7f98b9b). 

At least two parametrization of this test fail: test_progress_bar_max_val_check_interval

### Expected behavior

The CI fails when a test fails.

### Environment

PL master

### Additional context

As I always say, don't trust the CI. 

cc @tchaton @rohitgr7 @akihironitta @carmocca @borda

### PR DESCRIPTION
**Pull request by carmocca**
## What does this PR do?

See title

Fixes #12474

### Does your PR introduce any breaking changes? If yes, please list them.

None

cc @borda

### CODE REVIEW

File: .azure/gpu-tests-pytorch.yml [hunk 1]
```diff
@@ -36,7 +36,6 @@ pr:
       - "setup.cfg"  # includes pytest config
       - "requirements/lite/**"
       - "src/lightning_lite/**"
-      - "tests/tests_lite/**"
```
Comment (carmocca) [inline .azure/gpu-tests-pytorch.yml:39]: This is on purpose. Lite tests have nothing to do with PyTorch GPU tests

**Comment by carmocca** [comment]
Sorry for the review notification. Accidentally marked it as non-draft

**Comment by carmocca** [comment]
@awaelchli If you can help, this uncovered a failing Lite test: https://dev.azure.com/Lightning-AI/lightning/_build/results?buildId=112478&view=logs&j=8a08b103-7f3c-5f4b-1889-6a795acbb007&t=a889fbb9-184d-53b2-4806-a80daee53cd7&l=325: `strategies/test_fairscale_integration.py::test_fairscale_multi_process_checkpoint_state_consolidation[True-ddp_sharded-cpu]`

**Comment by awaelchli** [comment]
@carmocca was able to reproduce and fix locally. Should be resolved now. Let me know if there are any other failures  like this one popping up.

**Review by akihironitta — APPROVED** [review]

File: src/pytorch_lightning/strategies/fully_sharded.py [hunk 2]
```diff
@@ -197,8 +197,8 @@ def _setup_model(self, model: torch.nn.Module) -> FullyShardedDataParallel:
         log.detail(f"setting up `Fairscale FSDP` model with device id: {self.root_device.index}.")
 
         rank_zero_info(
-            "When using FairScale FSDP auto-wrap, make sure to initalize your model using trainer else"
-            " you will get an error.\ntorch.optim.Optimizer(self.trainer.model.parameters(), ...)"
+            "When using FairScale FSDP auto-wrap, make sure to initialize your model using trainer: "
```
Comment (awaelchli) [inline src/pytorch_lightning/strategies/fully_sharded.py:200]: This message will be removed in #15301

**Review by awaelchli — APPROVED** [review]

**Review by otaj — APPROVED** [review]

**Review by justusschock — APPROVED** [review]

**Comment by carmocca** [comment]
TPUs are failing, but there's an issue with the logs. Blocked by #15533

**Pull request merged**